csp-mixin/signals/azurevmOverview.libsonnet

```
@@ -0,0 +1,213 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: this.filteringSelector,
+    groupLabels: this.groupLabels,
+    instanceLabels: [],
+    aggLevel: 'instance',
+    discoveryMetric: {
+      azuremonitor: 'azure_microsoft_compute_virtualmachines_vmavailabilitymetric_average_count',
+    },
+    signals: {
+      instanceCount: {
+        name: 'Instance count',
+        description: 'Number of VM instances',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          azuremonitor: {
+            expr: 'count(sum by (resourceName) (azure_microsoft_compute_virtualmachines_vmavailabilitymetric_average_count{%(queriesSelector)s}))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+
+      vmAvailability: {
+        name: 'VM Availability',
+        description: 'Measure of availability of virtual machines',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          azuremonitor: {
+            expr: 'avg by(resourceName) (azure_microsoft_compute_virtualmachines_vmavailabilitymetric_average_count{%(queriesSelector)s})',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      top5CpuUtilization: {
+        name: 'Top 5 Instances - CPU Utilitization',
```

Can you update these titles please? 
```suggestion
        name: 'Top 5 Instances by CPU Utilitization',
```